### ISSUE TITLE
README file omits Open AI API key configuration 

### ISSUE THREAD

**Issue by specialkapa** [Wytamma/write-the#3]
Great job with this package. Appears to be very useful. My 2 cents on the README file. It appears to not describe how to point the package to an OpenAI API key. Perhaps I've missed something obvious in which case I apologise

**Comment by Wytamma** [comment]
Good point! I added it to the usage information -> https://github.com/Wytamma/write-the/blob/master/README.md#usage. Thanks for the feedback.
